Android.mk

```diff
@@ -58,6 +58,10 @@ ifeq "$(HYBRIS_FSTABS)" ""
 TARGET_VENDOR := "$(shell echo $(PRODUCT_BRAND) | tr '[:upper:]' '[:lower:]')"
 HYBRIS_FSTABS := $(shell find device/$(TARGET_VENDOR) -name *fstab* | grep -v goldfish)
 endif
+# Some devices are using subfolders
```

Comment:
Or are some devices withing subfolders? :) (that changes the logic below in device/*/*/$DEVICE vs device/*/$DEVICE/*

Reply:
changed wording :)